test/new-e2e/tests/apm/docker_test.go

```diff
@@ -88,7 +131,6 @@ func (s *DockerFakeintakeSuite) TraceAgentMetrics() {
 		}
 		metrics, err := s.Env().FakeIntake.Client().GetMetricNames()
 		assert.NoError(c, err)
-		s.T().Log("Got metric names", metrics)
```

Comment:
While I agree these logs can be spammy, they can also be super useful for debugging. Note that they only appear if we run the tests with the `-v` (verbose) option. I'd vote for keeping them unless there other reasons why you deleted them.

Reply:
I will add these back.